### ISSUE TITLE
"$__not_provided__$" when on option without a default

### ISSUE THREAD

**Issue by Wulfheart** [thettler/laravel-console-toolkit#8]
I would expect that not to be shown as I didn't define a default.

![image](https://user-images.githubusercontent.com/25671390/203080498-167ea735-0e3d-4a61-8dec-64ef8ab68289.png)

```php
<?php

namespace App\Console\Commands\Database;

use Illuminate\Console\Command;
use Thettler\LaravelConsoleToolkit\Attributes\Argument;
use Thettler\LaravelConsoleToolkit\Attributes\ArtisanCommand;
use Thettler\LaravelConsoleToolkit\Attributes\Option;
use Thettler\LaravelConsoleToolkit\Concerns\UsesConsoleToolkit;

#[ArtisanCommand(name: "create")]
class CreateDatabaseCommand extends Command
{
    use UsesConsoleToolkit;

    #[Argument]
    public string $project_name;

    #[Argument]
    public string $database_name;

    #[Option(description: "Memory in Mi")]
    public int $memory;
    #[Option(description: "CPU in millicores")]
    public int $cpu;
    #[Option(description: "Storage in Mi")]
    public int $storage;
    #[Option(description: "Replicas")]
    public int $instances = 1;

    public function handle(): void
    {
        
    }

}

```

**Comment by Wulfheart** [comment]
Nice! Thank you.

**Comment by thettler** [comment]
Should be fixed in version 0.1.1 :D
